.storybook/preview-head.html

```diff
@@ -2,7 +2,7 @@
 <link href="https://cdn.jsdelivr.net" rel="preconnect" crossorigin="anonymous">
 
 <!-- OUDS Web CSS -->
-<link href="https://cdn.jsdelivr.net/npm/@ouds/web/dist/css/ouds-web.min.css" rel="stylesheet" crossorigin="anonymous">
+<link href="https://cdn.jsdelivr.net/npm/@ouds/web/dist/css/ouds-web-bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
```

Comment:
Do we need to rely on `ouds-web-bootstrap.min.css` for Storybook as we're for now only will use it for ZeroHeight to show specific OUDS content?

Reply:
Yeah maybe we can use the default one for now and change it if needed after, I had in mind a use case that shouldn't happen.